app/Jobs/DatabaseBackupJob.php

```
@@ -682,6 +681,191 @@ private function getFullImageName(): string
+            $this->team->notify(new BackupFailed(
+                $this->backup,
+                $this->database,
+                $errorMsg,
+                $this->database->postgres_db
+            ));
+
+            throw $e;
+        } finally {
+            BackupCreated::dispatch($this->team->id);
+        }
+    }
```

PHPMD (2.15.0)

The method run_pgbackrest_backup() has a Cyclomatic Complexity of 19. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)